src/adapters/http/server.ts

```diff
@@ -0,0 +1,122 @@
+import Adapter from "../../lib/adapter.js";
+import GleeMessage from "../../lib/message.js";
+import http from "http";
+import { validateData } from "../../lib/util.js";
+import GleeError from "../../errors/glee-error.js";
+import * as url from "url";
+
+class HttpAdapter extends Adapter {
+  private res: any[] = [];
```

Comment:
This is very basic, you should this to cache connections, store the connection so that you can later distinguish when there are more than one HTTP server in the spec.